less/components/expander.less

```diff
@@ -21,6 +21,9 @@ html.no-js {
     font-weight: bold;
     text-transform: uppercase;
     padding: 2rem;
+    margin: 0;
+    font-size: inherit;
```

Comment:
This gets overriden by https://github.com/toolness/teach.webmaker.org/blob/639a89c0a366d91be0dc0029303dbcf1f31e3f1d/less/pages/clubs-toolkit.less#L13.  I'm guessing we can just have `font-size: 1.5rem` (or `1.6rem` to match default body copy size) to unify all `.expander-header` font sizes?